rfcs/000-rfc-lightweightRFCProcess.md

```diff
@@ -0,0 +1,87 @@
+* **Superseded**: The changes proposed on this RFC aren't in effect anymore, the document is kept for historical purposes and there is a new RFC that’s more current.
+
+### Approval
+The proposal should be posted with a date by which the author would like to see the approval decision to be made. How much time is given to comment depends on the size and complexity of the proposed changes. Anything between 1 week and 3 weeks seems like a reasonable timespan. Driving the actual decisions should follow the [lazy majority](https://cwiki.apache.org/confluence/display/KAFKA/Bylaws#Bylaws-Approvals) approach. If a discussion is still going strong once it’s approaching the initially set date, the author(s) might consider extending the time for discussion. This should happen explicitly and be announced on the RFC’s PR by updating the PR description accordingly.
+
+### When to write an RFC?
+Writing an RFC should be entirely voluntary. There is always the option of going straight to a pull request. However, for larger changes, it might be wise to de-risk the risk of rejection of the pull request by first gathering input from the community. Therefore it’s up to every member of our community to decide themselves when they want to reach for this tool.
+
+It’s encouraged to write an RFC for any major change. A major change might be:
+Addition of any major new feature or subsystem
+Changes that impact existing, public APIs. This includes Java APIs, but also things like GFSH commands and externally exposed protocols.
+
+### How to write an RFC?
+1. Copy the RFC template (in same folder as this document) and write your proposal! It's up to the author's discretion to decide which section in the template make sense for their proposal. It's recommended to cover the problem the proposal is solving, who it affects, how you’re proposing to solve it, and answers to frequently asked questions. Explicitly listing the goals will also make it easier to retrospect later on the proposal.
+2. Add your RFC to the geode/rfcs source directory, update to the next unique number. 
+3. Post a PR for your RFC putting [RFC] in the title. Make sure to state the deadline clearly in your PR. 
+The duration of the deadline is up to the author(s) and should depend on the size and complexity of the proposal. A reasonable time frame might be somewhere between a week and a month.
+4. Answer questions and concerns on the PR. Consider adding questions that get asked more than once to the FAQ section of the RFC.
+5. After the deadline for feedback has been reached summarize the consensus and your decision on the PR  thread. 
+    1. If the decision is to drop the proposal, the status should be updated to Dropped. 
+    2. If we are moving forward the status goes to *Active*
+    3. When there is a newer RFC that replaces this one the status goes to *Superseded* and the *Superseded By* gets updated with the number of the new RFC.
+    
+###Immutability & Errata
```

Comment:
Add white space between the ### and the title. In the preview it is shown as a normal string, not as a title.